src/EnergyPlus/UtilityRoutines.cc

```diff
@@ -1844,12 +1844,12 @@ namespace UtilityRoutines {
 
 
         if (UtilityRoutines::outputErrorHeader && err_stream) {
-            *err_stream << "Program Version," + VerString + ',' + IDDVerString + DataStringGlobals::NL;
+            *err_stream << "Program Version," << VerString << ',' << IDDVerString << '\n';
```

Comment:
What, [not](https://www.youtube.com/watch?v=GMqQOEZYVJQ) `std::endl`?

Reply:
https://www.youtube.com/watch?v=GMqQOEZYVJQ

Reply:
Oh I missed your "not" being a link :D